### ISSUE TITLE
Feedback on mini_ex4 by Rolf Holm

### ISSUE THREAD

**Issue by OorthianEmissary** [a9neh/Anines-mini_ex#7]
The bouncing donut in the grid is aestheticly pleasing. The Colorscheme fits together, as they are all pastel-ish. The donut gif is simple and easy to understand. Not too much is going on. Without having looked at the code, I guess it checks to see, if the gifs locations is either the full height, zero height and likewise with the width, and changes colour depending on that. 
To make it more interactive, the user could maybe have some say to the movement of the donut? The point of feedback, is to confirm that the user did something, and hopefully the right thing. The feedback as of right now, is only that the program works as I open it.

In conclusion I think it's a aestheticly pleasing piece with a lovely colourscheme.
